testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java

```diff
@@ -736,7 +738,7 @@ public void verifyFT22()
         InboxCallDetailRecord inboxCallDetailRecord = inboxCallDetailsDataService
                 .retrieve("callingNumber", 3000000000L);
         assertTrue(3000000000L == inboxCallDetailRecord.getCallingNumber());
-        assertTrue(123456789012345L == inboxCallDetailRecord.getCallId());
+        assertTrue(VALID_CALL_ID.equals(inboxCallDetailRecord.getCallId()));
```

Comment:
Wouldn't assertEquals be better here so if it doesn't we'd get expected and actual in the output?